va/validation-authority_test.go

```diff
@@ -867,7 +869,7 @@ func TestDNSValidationNoServer(t *testing.T) {
 }
 
 func TestDNSValidationOK(t *testing.T) {
-	stats, _ := statsd.NewNoopClient()
+	stats := mocks.NewStatter()
```

Comment:
It looks like you started using a mock statter in each test, but didn't add verification of any of the things on it? It would be better to leave it as a `NoopClient` in tests where we don't actually need the mock stuff.